package.json

```diff
@@ -45,30 +45,30 @@
     "test:preview": "astro preview --root tests/fixture"
   },
   "dependencies": {
-    "@astrojs/image": "^0.16.1",
-    "@tailwindcss/line-clamp": "^0.4.2",
-    "@typescript-eslint/eslint-plugin": "^5.56.0",
-    "@typescript-eslint/parser": "^5.56.0",
-    "astro": "^2.1.5",
+    "@astrojs/image": "^0.16.3",
+    "@typescript-eslint/eslint-plugin": "^5.57.0",
+    "@typescript-eslint/parser": "^5.57.0",
+    "astro": "^2.1.8",
     "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-astro": "^0.26.1",
     "eslint-plugin-jsx-a11y": "^6.7.1",
     "eslint-plugin-prettier": "^4.2.1",
     "prettier-plugin-astro": "^0.8.0",
     "prettier-plugin-tailwindcss": "^0.2.5",
-    "sharp": "^0.31.3",
-    "smartypants": "^0.1.6",
-    "tailwindcss": "^3.2.7"
+    "sharp": "^0.32.0",
+    "smartypants": "^0.1.6"
   },
   "devDependencies": {
     "@changesets/cli": "^2.26.1",
-    "@playwright/test": "^1.31.2",
-    "eslint": "^8.36.0",
+    "@playwright/test": "^1.32.1",
+    "eslint": "^8.37.0",
     "npm-run-all": "^4.1.5",
-    "prettier": "^2.8.6",
+    "prettier": "^2.8.7",
+    "tailwindcss": "^3.3.0",
     "typescript": "^5.0.2"
   },
   "peerDependencies": {
-    "eslint": ">=8"
+    "eslint": ">=8",
+    "tailwindcss": ">=3.3"
```

Comment:
`pnpm` has made me never trust peer dependencies 😅 This should work fine but keep an eye out for weird CI build issues if pnpm in Netlify builds doesn't pick this up right